cuda/private/toolchain_configs/dummy.bzl

```diff
@@ -0,0 +1,4 @@
+def _dummy_toolchain_config_impl(_ctx):
```

Comment:
Could you please rename the "dummy" to something like "disable" (also the filename). So that the purpose is made clear.

Reply:
Fixed.